src/CSSMap.js

```diff
@@ -0,0 +1,27 @@
+/**
+ *
+ * Maps the given properties to a unit type
+ *
+ * Currently only supports basic CSS methods - in order to support more advance CSS3 properties, such as
+ * transform, we will need to modify the way the properties array is handled and moved through the system. 
+ * In the case of CSS methods such as "transform: rotate(Xdeg)" - the mapping will need to be more advanced,
+ * possibly adding a prefix / suffix mapping.
+ *
+ * @param {array}  properties - Properties of the object to tween
+ * @returns {array} - The suffix mapped to the property name
+ *
+ */
+module.exports = function (properties) {
+    
+        var cssMap = { height: 'px', width: 'px', top: 'px', left: 'px', bottom: 'px', right: 'px', opacity: '' };
+        var mapping = {};
+    
+        for (var i = 0; i < properties.length; i++) {
+                if (properties[i] in cssMap) {
```

Comment:
A very good question. I guess the most appropriate course of action would be to throw an error. If we can't provide a CSS Tween for that given property, it probably shouldn't fail silently. 

However, having a quick look through the source for Tina I can't see any other instances of a thrown error. So I'm not sure what the site policy for error handling is. Any thoughts would be appreciated.